folly/ClockGettimeWrappers.cpp

```diff
@@ -23,7 +23,6 @@
 #include <time.h>
 
 #ifndef _WIN32
-#define _GNU_SOURCE 1
```

Comment:
Is this known not to be needed? Here and elsewhere.

Reply:
_GNU_SOURCE is already defined here: https://github.com/facebook/folly/blob/master/CMake/FollyCompilerUnix.cmake#L23

I removed as it throws a redefinition error. Defining _GNU_SOURCE to 1 is wrong anyway.